res/controllers/Pioneer-DDJ-SB2-scripts.js

```diff
@@ -0,0 +1,1187 @@
+var PioneerDDJSB2 = function() {};
+
+/*
+    Find the latest code at https://github.com/dg3nec/mixxx
+   
+    
+    This mapping for the Pioneer DDJ-SB2 was made by DG3NEC, Michael Stahl
+    Basing on DDj-SB for Mixxx 1.12 Joan Ardiaca Jové (joan.ardiaca@gmail.com),
+    basing on the work of wingcom (wwingcomm@gmail.com, https://github.com/wingcom/Mixxx-Pioneer-DDJ-SB).
+    which in turn was based on the work of Hilton Rudham (https://github.com/hrudham/Mixxx-Pioneer-DDJ-SR).
+    Just as wingcom's and Rudham's work, this mapping is pusblished under the MIT license.
+ */
+
+
+///////////////////////////////////////////////////////////////
+//                       USER OPTIONS                        //
+///////////////////////////////////////////////////////////////
+
+// If true the sync button blinks with the beat, if false led is lit when sync is enabled.
+PioneerDDJSB2.blinkingSync = true;
```

Comment:
Personally, I think this should be false by default, but I don't care too much either way because it is easily configurable.